gradle/dependencies.gradle

```diff
@@ -35,26 +35,27 @@ def versions = [
 ext.deps = [
         "versions": versions,
         "build": [
-                buildToolsVersion: '30.0.3',
-                compileSdkVersion: 31,
+                buildToolsVersion: '33.0.0',
+                compileSdkVersion: 33,
                 ci: 'true' == System.getenv('CI'),
                 minSdkVersion: 21,
                 targetSdkVersion: 30,
```

Comment:
I know we're limiting scope of changes but should we follow up with a bump in target as well?

Reply:
it only affects samples & it had further changes that i didn't want to tackle